### PR TITLE
Restore saving TrackingUrl from NG

### PR DESCRIPTION
**Pull request by narma**

### CODE REVIEW

File: app/async/workers/orders_puller.rb [hunk 1]
```diff
@@ -86,6 +86,8 @@ def update_shipments(shipments)
         attributes[:ship_address_attributes].merge!({country_id: country_id}) if country_id
 
         order.assign_attributes(attributes)
+        order.shipments.update_all({tracking: shipment['Tracking'],
```
Comment (Axblade) [inline app/async/workers/orders_puller.rb]: Why not to move it to shipment update section? https://github.com/FoxComm/spree_newgistics/blob/master/app/async/workers/orders_puller.rb#L97

**Comment by Axblade** [comment]
:+1:

**Comment by jmataya** [comment]
Looks good, but can you please add test coverage for this bug.

**Comment by narma** [comment]
My test is based on `orders_puller` test from #28, so any prospective conflicts should be minimal.

**Comment by jmataya** [comment]
Looks good.

**Pull request merged**